rule-engine/rule-engine-demos-test/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>ch.ivyteam.demo</groupId>
+  <artifactId>rule-engine-demos-test</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>iar-integration-test</packaging>
+  <dependencies>
+    <dependency>
+      <groupId>ch.ivyteam.demo</groupId>
+      <artifactId>rule-engine-demos</artifactId>
+      <version>[9.1.0-SNAPSHOT,)</version>
```

Comment:
I would use a specific version instead of maven ranges.

Reply:
Great catch, totally forgot to update this auto generated pom :)